application.yml

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
why is this config added on the main level, looks like a copy of `transmart-api-server/application.yaml`?

Reply:
you are right. it should not end up here.